### PR TITLE
install-photobooth: respect defined php version

### PR DESCRIPTION
**Pull request by Styne13**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)
- [x] Bug fix
- [ ] Documentation update
- [ ] Bug fix
- [ ] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)
- install-photobooth.sh: ~~Add composer and~~ modules depending on php version

#### Is there anything you'd like reviewers to focus on?
I installed photobooth on Rapberry Pi OS bookworm with the following command from the installation guide:

`
sudo bash install-photobooth.sh -username='<YourUsername>' -webserver='nginx'
`

It seems php modules gd, xml, zip and mbstring are not installed/activated correctly.
~~Also composer was missing, so the installation failed, so I like to fix it in this PR.~~

### CODE REVIEW

**Comment by andi34** [comment]
I think PHP 8.3 is the current stable now and installed by default with it's 8.3 packages.

File: install-photobooth.sh [hunk 1]
```diff
@@ -59,10 +59,10 @@ COMMON_PACKAGES=(
     'gphoto2'
     'libimage-exiftool-perl'
     'nodejs'
-    'php-gd'
-    'php-xml'
-    'php-zip'
-    'php-mbstring'
+    "php${PHP_VERSION}-gd"
```
Comment (andi34) [inline install-photobooth.sh:63]: libapache2-mod-php${PHP_VERSION}

Where needed

File: install-photobooth.sh [hunk 2]
```diff
@@ -499,6 +499,10 @@ function common_software() {
     if [ "$NEEDS_NPM_CHECK" = true ]; then
         check_npm
     fi
+
+    # Install php composer
+    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```
Comment (andi34) [inline install-photobooth.sh]: composer exists inside the bin/ directory of Photobooth

https://github.com/PhotoboothProject/photobooth/blob/dev/bin/composer

Comment (Styne13) [reply install-photobooth.sh]: Ok, thought I also got an error during `npm run build`

**Comment by Styne13** [comment]
Hope I'm not bothering you with this PR. :)
This is a great project and you are doing a good job.

Think you are right with the default version of PHP in bookworm. :+1:

**Review by dadav — APPROVED** [review]
Looks good to me

**Review by andi34 — COMMENTED** [review]
Thanks

**Pull request merged**